

#global testerror "Err0594"
func arithmeticMean(numbers: f32...)
{
}

#test
{
	arithmeticMean(true, 2)
}